Classes/TBODeveloperOverlayLogger/TBODeveloperOverlayFilterableLogFormatterProtocol.h

```diff
@@ -34,6 +34,16 @@
 
 - (NSRegularExpression *_Nullable)regularExpressionForLogLevel:(NSString *_Nonnull)logLevel;
 
+@optional
+
+/**
+ *  Use this method to define what is a single log message. This way you can have one log message span multiple lines.
+ *  If not defined, one message is expected to be one line.
+ *
+ *  @return the NSRegularExpression that matches exactly one log message.
+ */
+- (NSRegularExpression *_Nullable)logMessageRegularExpression;
```

Comment:
This is the magic here. Basically a log formatter now can define where a log message starts and where it ends. This is done with a regEx that always matches a full log message.